AndroidSDKCore/src/main/java/com/leanplum/internal/LeanplumEventDataManager.java

```diff
@@ -48,53 +48,58 @@
  * @author Anna Orlova
  */
 public class LeanplumEventDataManager {
+
+  private static LeanplumEventDataManager instance;
+
   private static final String DATABASE_NAME = "__leanplum.db";
   private static final int DATABASE_VERSION = 1;
   private static final String EVENT_TABLE_NAME = "event";
   private static final String COLUMN_DATA = "data";
   private static final String KEY_ROWID = "rowid";
 
-  private static SQLiteDatabase database;
-  private static LeanplumDataBaseManager databaseManager;
-  private static ContentValues contentValues = new ContentValues();
+  private SQLiteDatabase database;
+  private LeanplumDataBaseManager databaseManager;
+  private ContentValues contentValues = new ContentValues();
+  private boolean sendErrorLogs = false;
```

Comment:
Any reason for "sendErrorLogs" rename?

Reply:
Just to make it consistent, nothing else.